test/fluree/db/query/sparql_test.cljc

```diff
@@ -84,7 +84,21 @@
           {:keys [where]} (sparql/->fql query)]
       (is (= [{"@id"           "?person"
                "person:handle" "jdoe"}]
-             where))))
+             where)))
+    (let [query  "SELECT ?person WHERE {?person person:handle \"Los Angeles\"@en .}"
+          {:keys [where]} (sparql/->fql query)]
+      (is (= [{"@id" "?person",
+               "person:handle" {"@value" "Los Angeles", "@language" "en"}}]
+             where)
+          "lang literal"))
+    (let [query "SELECT ?person
+                 WHERE {?person person:birthday \"2011-01-10T14:45:13.815-05:00\"^^xsd:dateTime .}"
+          {:keys [where]} (sparql/->fql query)]
+      (is (= [{"@id" "?person",
+               "person:birthday"
+               {"@value" "2011-01-10T14:45:13.815-05:00", "@type" "xsd:dateTime"}}]
+             where)
+          "datatype literal")))
```

Comment:
Would be good to move these into their own descriptive `testing` form. They're moving away from "simple triple" :)

Reply:
The "simple triple" refers more to the grammar classification of this type of expression.  These test cases just confirm that all the different types of RDF literal terms are translated correctly.